react/components/atoms/lightbox/lightbox.stories.js

```diff
@@ -9,7 +10,7 @@ storiesOf("Atoms", module)
     .add("Lightbox", () => {
         const uri = text(
             "URI",
-            "https://sandbox.platforme.com/api/compose?brand=dummy&model=dummy"

```

Comment:
why take the brand off?

Reply:
I still don't know why but the `text` function from storybook was encoding the `&` in the URI